src/QuoteProcessor.php

```diff
@@ -87,20 +87,26 @@ public function process(AbstractStringContainer $opener, AbstractStringContainer
     /**
      * Create a double-quote processor
      *
+     * @param string $opener
+     * @param string $closer
+     *
      * @return QuoteProcessor
      */
-    public static function createDoubleQuoteProcessor(): self
+    public static function createDoubleQuoteProcessor(string $opener, string $closer): self
```

Comment:
Can we add default values for these two parameters?  For example:

```
public static function createDoubleQuoteProcessor(string $opener = Quote::DOUBLE_QUOTE_OPENER, string $closer = Quote::DOUBLE_QUOTE_CLOSER): self
```

If you can do this for both static methods then we can introduce this as a backward-compatible change :)